src/components/Login/Login.js

```diff
@@ -14,9 +14,23 @@ class Login extends Component {
     this.setState({[e.target.name]: e.target.value})
   }
 
-  fetchUserData = (e) => {
+  checkUserData = (e) => {
     e.preventDefault();
-    fetch('https://rancid-tomatillos.herokuapp.com/api/v1/login',
+    this.fetchUserData()
+      .then(response => {
+        if(response.ok === true) {
+          this.fetchUserData()
+            .then(response => response.json())
+            .then(data => console.log(data))
+        } else {
+          alert("bad credentials");
```

Comment:
This bit seems a bit redundant making the fetch call twice. We should be able to get away with removing one of those since it's returning the same thing twice.